app/api/task_sheet/task_sheet.tsx

```diff
@@ -0,0 +1,64 @@
+// Should return all task_sheets student/adult is part of
+// request will send you a key (I guess we don't check if key is parent of child) and you will check the columns for student_id and adult_ids and if it’s in it we  return it.
+
+// **Tasks Table**
+// | Column | Type | 
+// | task_id | integer | 
+// | student_id | integer |  
+// | adult_ids | json |  
+// | task_name | string |
+
+
+import { NextResponse, NextRequest } from 'next/server';
+
+const adults_json = '{ "adult_ids" : [' +
+'{ "id": 1 },' +
+'{ "id":2 } ]}'; 
+
+const tasks: Task[] = [
+    { task_id: 1, student_id: 1, adult_ids: adults_json, task_name: "homework1"},
+    { task_id: 2, student_id: 2 , adult_ids: adults_json,task_name: "homework2"}
+];
+
+function findTasks(id: number, tasks_local: Task[]): Task[] {
+    var foundTasks: Task[] = [];
+
+
+    for (let i = 0; i < tasks_local.length; i++) {
+        let adults_object = JSON.parse(tasks_local[i].adult_ids);
+
+        // Function to check if a number exists in the array
+        const checkIdExists = (idToCheck: number) => {
+            return adults_object.adult_ids.some((item: { id: number }) => item.id === idToCheck);
+        };
+        if (tasks_local[i].student_id === id || checkIdExists(id)) {
```

Comment:
I know this is necessery now because we aren't querying but in future "tasks_local[i].student_id === id" won't be needed because of the query. Noting it here so we can remove it later. Please add a TODO comment so we can come back to this.